src/main/scala/com/high-performance-spark-examples/dataframe/HappyPandas.scala

```diff
@@ -5,19 +5,18 @@ package com.highperformancespark.examples.dataframe
 
 import org.apache.spark._
 //tag::sparkSQLImports[]
-import org.apache.spark.sql.{DataFrame, SQLContext, Row}
-import org.apache.spark.sql.catalyst.expressions.aggregate._
+import org.apache.spark.sql.{DataFrame, SQLContext}
```

Comment:
I think here for demonstration purposes we want to keep the full list of imports (even though we don't need all of them for the following example code).